### ISSUE TITLE
Preprocessor has problem comparing numbers with the `l` suffix.

### ISSUE THREAD

**Issue by MarekKnapek** [PhilippRados/wrecc#4]
The `l` is lowercase `L`.
Steps to reproduce:

```
$ cat program.c
```

```
#if defined __cplusplus && __cplusplus >= 199711l
#endif

int main(void)
{
}
```

```
$ wrecc -D NDEBUG program.c
```

```
error: found trailing tokens after preprocessor expression
|  --> in program.c:1:49
|
1 #if defined __cplusplus && __cplusplus >= 199711l
|                                                 ^
1 error generated.
```

**Comment by PhilippRados** [comment]
Yes the scanner currently doesn't support annotated numeric-literals like (0x01 or 2L)

**Comment by PhilippRados** [comment]
just implemented it, so it should work in release 0.3.0 which will be out soon
